Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Made by Pato Grone for SS14",
+  "copyright": "Made by Pato Grone and Jonathan Argo for SS14",
```

Comment:
Made by Pato Grone for SS14, Wielded states by Jonathan Argo

Reply:
Ah, of course, good idea, thanks!